das/checkpoint_store.go

```diff
@@ -2,7 +2,7 @@ package das
 
```

Comment:
We might want to rename this file to just `checkpoint`

Reply:
renamed. Thank you